pkg/utils/utils.go

```diff
@@ -675,3 +675,15 @@ func GetSvcSnapshotName(snapshotId string) (string, error) {
 	}
 	return snapshotIdParts[len(snapshotIdParts)-1], nil
 }
+
+func GetRepo(image string) string {
```

Comment:
Please document your algorithm here in a comment.
If I understand it correctly, you are removing the "/" and then adding the "/" back in.  Why don't you use LastIndex for "/" and just split there?  You are splitting the string all over again each time through the loop.  For this size it's not a big deal but this is an N^2 algorithm and a bad habit.  You should split the strings outside the loop and keep the len
Please do not use "i"

Reply:
Changed to use strings.LastIndex to find the last '/'.

Reply:
Please leave a comment about what is being done here.  What is the expected format of the image name?  Why are you sure the last slash will be the slash you should split at?